en/lessons/basics/modules.md

```diff
@@ -193,7 +193,7 @@ import List, only: :macros
 
 ### `require`
 
-Although used less frequently `require/2` is nonetheless important.  Requiring a module ensures that it is compiled and loaded.  This is most useful when we need to access a module's macros:
+You could use `require` to tell Elixir you're going to macros from other module. The slight difference with `import` is that it allows using macros, but not functions from the specified module:
```

Comment:
It feels like a verb is missing before `macros` in `you're going to macros`..

Reply:
I think we want to add in "use"